galasa-maven-plugin/src/main/java/dev/galasa/maven/plugin/DeployTestCatalog.java

```diff
@@ -100,21 +105,36 @@ public void execute() throws MojoExecutionException, MojoFailureException {
 
         URL testcatalogUrl = calculateTestCatalogUrl(bootstrapProperties, this.testStream, this.bootstrapUrl);
 
-        checkGalasaAccessTokenIsValid(this.galasaAccessToken); 
-
-        String jwt = getAuthenticatedJwt(this.authFactory, this.galasaAccessToken, this.bootstrapUrl) ;
+        String jwt = null ;
+        // For now, if no galasa token is supplied, that's ok. It's optional.   
+        if (this.galasaAccessToken==null || this.galasaAccessToken.isEmpty()) {
```

Comment:
The if statement could be condensed to just make sure that if !=null and it not empty then call the getAuthenticatedJwt